### PR TITLE
Move common cd targets to common Makefile

### PR DESCRIPTION
**Pull request by Brian-McM**

### CODE REVIEW

**Review by doublek — COMMENTED** [review]
Nice change. This looks good and I've noticed these particular targets copied all over and should trim some of our other Makefiles. Added one comment suggestion and one question.

File: Makefile.common [hunk 1]
```diff
@@ -627,6 +627,76 @@ docker-compress:
...
+endif
+
+## push multi-arch manifest where supported
+push-manifests: imagetag  $(addprefix sub-manifest-,$(call escapefs,$(PUSH_MANIFEST_IMAGES)))
+sub-manifest-%:
+	# Docker login to hub.docker.com required before running this target as we are using $(DOCKER_CONFIG) holds the docker login credentials
+	# path to credentials based on manifest-tool's requirements here https://github.com/estesp/manifest-tool#sample-usage
+	docker run -t --entrypoint /bin/sh -v $(DOCKER_CONFIG):/root/.docker/config.json $(CALICO_BUILD) -c "/usr/bin/manifest-tool push from-args --platforms $(call join_platforms,$(VALIDARCHES)) --template $(call unescapefs,$*:$(IMAGETAG))-ARCH --target $(call unescapefs,$*:$(IMAGETAG))"
+
+cd-common:
```
[the remark this answers is not in the document]
Comment (Brian-McM) [reply Makefile.common:690]: Good thinking, I'll do that.

File: Makefile.common [hunk 2]
```diff
@@ -627,6 +627,76 @@ docker-compress:
...
+ifndef CONFIRM
+	$(error CONFIRM is undefined - run using make <target> CONFIRM=true)
+endif
+ifndef BRANCH_NAME
+	$(error BRANCH_NAME is undefined - run using make <target> BRANCH_NAME=var or set an environment variable)
+endif
```
[the remark this answers is not in the document]
Comment (Brian-McM) [reply Makefile.common:696]: That's a good call, I'll do that.

**Review by doublek — APPROVED** [review]
LGTM

**Pull request merged**